lightfm-php-test/composer.json

```
@@ -0,0 +1,19 @@
+{
+    "name": "test/lightfm-php-test",
+    "description": "Test project for lightfm-php package installation",
+    "type": "project",
+    "require": {
```

Biome (1.9.4)

[error] The key require was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)